package.json

```diff
@@ -19,8 +19,9 @@
 		"test": "pnpm --filter @gradio/client build && vitest dev --config .config/vitest.config.ts",
 		"test:run": "pnpm --filter @gradio/client build && vitest run --config .config/vitest.config.ts",
 		"test:node": "TEST_MODE=node pnpm  vitest run --config .config/vitest.config.ts",
-		"test:browser": "pnpm --filter @gradio/app test:browser:full",
+		"test:browser": "pnpm --filter @gradio/app test:browser",
 		"test:browser:full": "run-s build test:browser",
+		"test:browser:verbose": "GRADIO_TEST_VERBOSE=true pnpm test:browser",
```

Comment:
I silence the python child process's stderr and stdout by default because it is incredibly noisy and interferes with the tests but if something goes wrong we really need to be able to see all of the logs that the child process is generating. This script enable 'verbose mode' which will print anything that is logged.